tests/test_wellbore_marker_frame.py

```diff
@@ -194,8 +146,8 @@ def test_get_interpretation_object(example_model_and_crs):
     zs = mds - elevation
     source_dataframe = pd.DataFrame({
         'MD': mds,
-        'X': [1, 2, 3],
-        'Y': [1, 2, 3],
+        'X': [150, 165, 180],
```

Comment:
Better to use floating point numbers, 150.0 etc., in case behaviour changes with integers.